nengo_viz/static/viz_sim_control.js

```diff
@@ -9,17 +9,17 @@
 VIZ.SimControl = function(div, args) {
     var self = this;
 
-    div.classList.add('sim_control');    
+    div.classList.add('sim_control');
     this.div = div;
     div.style.position = 'fixed';
-    //TODO: Is this the method we want to use for keeping simcontrol on top 
+    //TODO: Is this the method we want to use for keeping simcontrol on top
     //or do we want to dynamically raise it's zindex when other elements have theres increased
-    div.style.zIndex = 99999999; 
+    div.style.zIndex = 1040;
```

Comment:
Every time you click on something, the max_zindex increases.  So if you click on a 1000 things, items will start showing up on top of the modal.  :(

https://github.com/nengo/nengo_viz/blob/master/nengo_viz/static/viz.js#L82